src/plugins/intel_cpu/src/transformations/tpp/x64/pass/fuse_tpp_to_equations.cpp

```diff
@@ -82,7 +82,15 @@ bool FuseTPPToEquations::fuse_from_root(const NodePtr& root, const std::shared_p
         kv.second = equation;
     replace_nodes(m, {}, node_replace_map);
     for (const auto& in : equation->inputs()) {
-        ov::snippets::lowered::PortDescriptorUtils::set_port_descriptor(in, root_subtensor);
+        auto subtensor = root_subtensor;
+        if (in.get_partial_shape().size() < root_subtensor.size()) {
+            subtensor.resize(in.get_partial_shape().size());
```

Comment:
First of all, order of dimensions in shape and subtensor is the same. Since the kernel processes last dimensions, subtensor should cover in the end of shape:

```
loop_depth = 2
shape = [a, b, c, d]
subtensor = [c, d]
```

Your current solution set upper (first) dimensions to subtensor: `[a, b]`.

Then please update the code for output subtensor. It has the same problem with `2` value.

Reply:
> Your current solution set upper (first) dimensions to subtensor: [a, b].

Fixed to take `[c, d]`

> Then please update the code for output subtensor. It has the same problem with 2 value.

Applied fix for outs as well